sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -858,72 +841,6 @@ Exceptions:
 
 |===
 
-:sycl-kernel-function: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sycl-kernel-function
-
-Table {counter: tableNumber}. Member functions of the `command_graph` class (executable graph update).
-[cols="2a,a"]
-|===
-|Member function|Description
-
-|
-[source, c++]
-----
-void
-update(const command_graph<graph_state::modifiable>& graph);
```

Comment:
There's a section on `update` on line 534, do we want to move that too?

Also, line 386 in the API overview is the update declaration that can be removed.

Reply:
Good catch. I've (re-)moved this part on updating a graph.